src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,30 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Rectangle rectangle = (Rectangle) o;
+
+        if (!(this.length.equals(rectangle.length) && this.width.equals(rectangle.width)
```

Comment:
if (!(this.length.equals(rectangle.length) && this.width.equals(rectangle.width)
                && this.color.equals(rectangle.color))) {
            return false;
        }

        return true;
-->
return this.length.equals(rectangle.length)
&& this.width.equals(rectangle.width)
&& this.color.equals(rectangle.color);